modules/serverless-rn-1-32-0.adoc

```diff
@@ -0,0 +1,65 @@
+
+[id="new-features-1-32-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.11.
+* {ServerlessProductName} now uses Knative Eventing 1.11.
+* {ServerlessProductName} now uses Kourier 1.11.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.11.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.11.
+* The `kn func` CLI plug-in now uses `func` 1.13.
+
+* Serverless Logic, which is available as a Technology Preview (TP) feature, has been updated.
++
+See link:https://openshift-knative.github.io/docs/docs/latest/serverless-logic/about.html[the Serverless Logic documentation] for usage instructions.
+
+* You can configure the {FunctionsProductName} readiness and liveness probe settings for the user-container and the queue-proxy container.
+
+* {FunctionsProductName} now supports {pipelines-shortname} versions `1.10` and `1.11`. The older versions of {pipelines-shortname} are no longer compatible with {FunctionsProductName}. 
+
+* {FunctionsProductName} now supports Go function container and it is now available as a Technology Preview (TP) feature.
+
+* The Knative TLS authentication feature for internal traffic is now deprecated. The functionality with the `internal-encryption` configuration flag is no longer available.
+
+* The secret filtering is enabled by default on the {ServerlessOperatorName} side. An environment variable, `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID=true`, is added by default to the `net-istio` and `net-kourier` controller pods.
+
+* The `domain-mapping` and `domain-mapping-webhook` deployments functionality in the `knative-serving` namespace is now removed.
+
+* If you have set `spec.config.domain` in the KnativeServing Custom Resource (CR), the default external domain will no longer auto-populate the config-domain ConfigMap in the `knative-serving` namespace. You will now need to configure it manually to ensure accurate domain settings. 
+
+* You can now use the gRPC health probe for `net-kourier` deployments. The `net-kourier-controller` now uses a standard Kubernetes gRPC health probe for both readiness and liveness, replacing its previous use of exec and custom commands. This simplifies the process and aligns it with Kubernetes standards. Additionally, the timeoutSeconds value has been adjusted from 100 milliseconds to 1 second to ensure more reliable probe responses.
+
+* The new trigger filters feature is now available as a Technology Preview. The new trigger filters are now enabled by default. It allows users to specify a set of filter expressions, where each expression evaluates to either true or false for each event.
+
+* Knative Eventing now offers support for data in transit encryption (Eventing TLS) as a developer preview. You can configure Knative Eventing components to encrypt data in-transit by exposing HTTPS addresses and accepting user-provided CA trust bundles. 
```

Comment:
```suggestion
* Knative Eventing now offers support for data in transit encryption (Eventing TLS) as a developer preview. You can configure Knative Eventing components to expose HTTPS addresses as well as add user-provided CA trust bundles to clients. 
```

Reply:
The sentence `encrypt data in-transit by exposing HTTPS` doesn't align well

Reply:
Okay, noted!